tests/tests_pytorch/serve/test_servable_module_validator.py

```diff
@@ -31,8 +30,7 @@ def configure_response(self):
         return {"output": [0, 1]}
 
 
-@RunIf(min_python="3.9")  # flaky on Python 3.8
-@pytest.mark.flaky(reruns=3)
+@pytest.mark.skip(reason="test is too flaky in CI")  # todo
```

Comment:
why not the xfail? otherwise we can just delete the test :)

Reply:
A test that is flaky with a probability P will become a flaky test with probability 1-P when converting it to xfail. The priority of this PR was to unblock other PRs who are frequently affected by this. Feel free to delete the test, but marking it as xfail would only work against us so please don't do it.

Reply:
sure, but the meaning of `xfail(strict=False` means it can fail or not and the pytest is happy